client/signup/steps/woocommerce-install/select-site/index.tsx

```diff
@@ -45,15 +44,18 @@ export default function SelectSite( props: WooCommerceInstallProps ): ReactEleme
 
 	const [ selectedSiteId, setSelectedSiteId ] = useState();
 
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, selectedSiteId ) );
+
 	const dispatch = useDispatch();
 
 	useEffect( () => {
-		if ( ! selectedSiteId ) {
+		if ( ! siteSlug ) {
 			return;
 		}
-		dispatch( submitSignupStep( { stepName: 'select-site' }, { siteId: selectedSiteId } ) );
+
+		dispatch( submitSignupStep( { stepName: 'select-site' }, { siteSlug } ) );
 		goToStep( 'confirm' );
-	}, [ selectedSiteId, goToStep, dispatch ] );
+	}, [ siteSlug, goToStep, dispatch ] );
```

Comment:
Was attempting to reduce confusion around siteId/slug but this didn't seem to matter too much in the end.

Reply:
It matters for the Hero Flow but don't think it does for `woocommerce-install` 🤔 
We might create a Hero Flow url using an old site slug, then after going through checkout the site slug changes (because of a domain purchase), which invalidates the site slug. That's why sometimes we care about site ID.
But I think for woop you're never coming straight from a domain purchase right?

Reply:
Maybe one day but I doubt it at this point (steps aren't really designed to be interchangeable like I expected they would be.)